lib/index.js

```diff
@@ -152,3 +169,28 @@ function stringify(obj) {
   // Events will be silently dropped if you do not stringify
   return JSON.stringify(ret);
 }
+
+/**
+ * Get the correct device type from the track payload.
+ * This function is required to support backwards compatibility
+ * with the functionality of this integration prior to v. 1.1.3
+ * 
+ * @api private
+ * @param {Track} track
+ * @return {String} deviceType
+ */
+
+function getDeviceType(track) {
+  var deviceType;
+  var deviceInfo = track.device() || {};
+
+  if (deviceInfo.type) {
+    deviceType = deviceInfo.type.toLowerCase();
+  } else if (contains(track.library(), 'ios')) {
+    deviceType = 'ios';
+  } else {
+    deviceType = 'android';
```

Comment:
I put some comments in the top of the function. Do you think we need more context?